app/assets/stylesheets/landing.scss

```diff
@@ -0,0 +1,172 @@
+nav {
+  border-radius: 3%;
+  height: 5em;
+}
+
+.medium {
+  background: url("http://i.imgur.com/CShEgXJ.jpg") no-repeat;
+  background-size: cover;
+  display: block;
+  height: 45em;
+  width: 100%;
+}
+
+.row-break {
+  height: 4em;
+  width: 100%;
+}
+
+.cat-heading::before,
+.cat-heading::after {
+  border-bottom: 1px solid #b2beb5;
+  -moz-box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+  content: " ";
+  display: block;
+  height: 0;
+  position: absolute;
+  top: 50%;
+  width: 40%;
+}
+
+.card-desc > span {
+  color: #000;
+  font-size: 0.9em;
+}
+
+.card-desc > span.price {
+  color: #338574 !important;
+}
+
+.card,
+nav {
+  box-shadow: none !important;
```

Comment:
!important should not be used<br>Properties should be ordered -moz-box-shadow, -webkit-box-shadow, box-shadow